Doc/library/colorsys.rst

```diff
@@ -37,6 +38,20 @@ The :mod:`colorsys` module defines the following functions:
    Convert the color from YIQ coordinates to RGB coordinates.
 
 
+.. function:: rgb_to_yuv(r, g, b)
+
+   Convert the color from RGB coordinates to YUV coordinates.
+
+   .. versionadded:: 3.11
+
+
+.. function:: yuv_to_rgb(y, i, q)
+
+   Convert the color from YUV coordinates to RGB coordinates.
+
+   .. versionadded:: 3.11
```

Comment:
Would you mind to document the change in Doc/whatsnew/3.11.rst as well?

Reply:
I've given it a shot - let me know if it look all right.